server/database/dbBuild.js

```diff
@@ -4,8 +4,7 @@ const dbConnection = require("./dbConnection")
 const sql = fs.readFileSync(`${__dirname}/dbBuild.sql`).toString()
 
 dbConnection.query(sql, (err, res) => {
-    if (err) {
-        throw err
-    }
-    console.log("response", res)
+  if (err) {
+    throw err
+  }
```

Comment:
I think this console is important but you could change it to "DB connected successfully"